lib/util/miq_apache/miq_apache.rb

```diff
@@ -125,7 +125,7 @@ def self.status(full = true)
     end
 
     def self.version
-       MiqUtil.runcmd("rpm -qa --queryformat '%{VERSION}' httpd")
+      MiqUtil.runcmd("rpm -qa --queryformat '%{VERSION}' #{MiqApache.package_name}")
```

Comment:
Yea, this belongs in linux_admin.
Would be very cool to have the scl concept in there in general.
But it is probably out of scope for this release